src/FakeItEasy/Messages/ThreePlaceholderMessage.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace FakeItEasy.Messages
+{
+    internal struct ThreePlaceholderMessage

```

Comment:
Ah, yes. It was used before I removed the unused resource. I could remove it, but on the other hand, we might need it later... I don't mind either way.